### ISSUE TITLE
.../IO/Socket/SSL.pm:1177: global error: Undefined SSL object

### ISSUE THREAD

**Issue by wltjr** [noxxi/p5-io-socket-ssl#109]
Hello, I am seeking information on the following output. Searching via [google](https://www.google.com/search?q="IO%2FSocket%2FSSL.pm%3A1177%3A"+global+error%3A+Undefined+SSL+object) there are a few links to issues/bugs that mention that very same output, most are using imapsync, I am using [assp](https://sourceforge.net/projects/assp/). It does not seem to affect operation, but it does not seem like good output. What is undefined? Is that anything to be concerned about or just generic output, or some means to fix what ever is causing that output. Thank you for any information that can be provided!

**Comment by wltjr** [comment]
Sure no problem, I will bring it up on assp mailing list. I just wanted to get some background information before hand. Thank you for confirming it is a usage issue, that is very helpful!

**Comment by mendel5** [comment]
Hi @noxxi , maybe the issue at https://github.com/imapsync/imapsync/issues/293 provides more details about the error message.

I think @OldGrumpy-de can provide you with logs (see his comment in the linked issue).

**Comment by ratijas** [comment]
I am too, getting this very same error, but during `git send-email` flow. Last time I set up git-email, I managed to resolve the issue somehow by installing missing modules. I don't remember the solution though. Anything's wrong with the way I call `$ git send-email patches/` ? :-)

FWIW, here's the relevant git config:

```
sendemail.smtpuser=me@ratijas.tk
sendemail.smtpserver=smtp.yandex.ru
sendemail.smtpencryption=ssl
sendemail.smtpserverport=465
```

Similar settings work fine on android (GMail app)
